frontend/server/src/main/java/com/amazonaws/ml/mms/wlm/ModelManager.java

```diff
@@ -193,4 +200,8 @@ public boolean scaleRequestStatus(String modelName) {
     public void submitTask(Runnable runnable) {
         wlm.scheduleAsync(runnable);
     }
+
+    public Set<String> getStartupModels() {
+        return startupModels;
```

Comment:
If you use a list, you can access it by index.

Reply:
I use a set to make it easy for show commands.